app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,20 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
 
+  TAG_EVENT_TYPE    = "miq.event_type".freeze # required by fetch
```

Comment:
It's referring to the Tag on the Hawkular event, so I think it makes sense in this situation.  I could rename to HAWKULAR_TAG_EVENT_TYPE but that seems verbose.